matrixbot/matrix.py

```diff
@@ -32,12 +32,16 @@ def __init__(self, settings):
         self.room_ids = settings["matrix"]["rooms"]
         self.domain = self.settings["matrix"]["domain"]
         self.only_local_domain = self.settings["matrix"]["only_local_domain"]
+        self.super_users = settings["matrix"]["super_users"]
 
         self.subscriptions_room_ids = settings["subscriptions"].keys()
         self.revokations_rooms_ids = settings["revokations"].keys()
         self.allowed_join_rooms_ids = filter(lambda x: x != 'default', settings["allowed-join"].keys())
         self.default_allowed_join_rooms = settings["allowed-join"]["default"]
 
+        self.enable_list_rooms_commands = settings["commands"]["list-rooms"]["enable"]
```

Comment:
Could you add an example for this in the  `matrix-bot.cfg.example` ?

Reply:
Sure. Added a minimal configuration working with the feat.